src/itemdef.cpp

```diff
@@ -543,7 +543,7 @@ class CItemDefManager: public IWritableItemDefManager
 					request = m_get_clientcached_queue.pop();
 
 			m_get_clientcached_queue.pushResult(request,
-					createClientCachedDirect(request.key, gamedef));
+					createClientCachedDirect(request.key, (Client *)gamedef));
```

Comment:
`gamedef` should be `client`. This function is called only from the `Game` object, and it already supplies the client as a parameter.